app/workers/breaker_status.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
Not for this PR, and totally but just a general note that our workers could use some naming cleanup. Most have a noun/verb combo, some have a `_job` suffix with a noun that doesn't indicate what they do.

Reply:
Ah yeah. What would an appropriate name be here? `BreakerStatusJob` for class and `breaker_status_job.rb` for file?

Reply:
`ExportBreakerStatus` maybe? `BreakerStatusJob` works, too, but I don't know if it gives as much context as it could. Not sure which convention is preferable, honestly.